### PR TITLE
All news functionality

### PR DESCRIPTION
**Pull request by Karyum**
rel to #102

### CODE REVIEW

File: src/queries/find_all_news.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { mockNews } = require('../../database/news_schema.js');
+
+const findAllNews = (hbs, response) => {
+  mockNews.find((error, news) => {
+    if (error) response.render('not_found')
+    else {
+      response.render('all_news', {
+        SingleNews: news.shift(),
```
Comment (Suuuuha) [inline src/queries/find_all_news.js]: why do you render a SingleNew?

File: src/queries/find_all_startups.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { mockCollection } = require('../database/startup_schema.js');
+const { mockCollection } = require('../../database/startup_schema.js');
 
 const findAllStartupsStartups = (options, hbs, response) => {
```
Comment (Suuuuha) [inline src/queries/find_all_startups.js]: is the 'options' necessary in here?

File: src/routes/results.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const findAllStartups = require('../find_all_startups.js');
+const findAllStartups = require('../queries/find_all_startups.js');
```
Comment (Suuuuha) [inline src/routes/results.js]: this should be changed to 'actions' when we move all functionality out of the routes

File: src/queries/find_all_news.js (hunk 1 above)
Comment (Karyum) [inline src/queries/find_all_news.js]: Yeah for styling reasons

File: src/queries/find_all_startups.js (hunk 2 above)
Comment (Karyum) [inline src/queries/find_all_startups.js]: yup its connected to the results page route

File: src/routes/results.js (hunk 3 above)
Comment (Karyum) [inline src/routes/results.js]: @Suuuuha 
Done cheers :+1:

**Comment by Karyum** [comment]
@esraajb I don't think @Suuuuha will be able to code review any time soon do you prefer to review the code before @Suuuuha  does?

File: src/actions/find_all_news.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const { mockNews } = require('../../database/news_schema.js');
+
+const findAllNews = (hbs, response) => {
```
Comment (esraajb) [inline src/actions/find_all_news.js]: hbs?

File: tests/server/index.js [hunk 5]
```diff
@@ -119,6 +119,15 @@ tape('Test for results', (t) => {
...
     t.equal(res.text.includes(html), true, 'Should render the results page');
     t.end();
+  })
+})
+
+tape('Test news section page', (t) => {
+  let html = '<p>at the end mario got the lazy plumber to fix the water cooler</p>';
+  supertest(server).get('/allNews').end((err, res) => {
+    t.error(err, 'No Error');
+    t.ok(res.text.includes(html), 'All events should be rendered');
```
Comment (esraajb) [inline tests/server/index.js]: all events?

**Comment by Karyum** [comment]
@esraajb changes were made :+1:

**Pull request merged**